### DISCUSSION TITLE
This user account is blocked

### DISCUSSION THREAD

**Discussion by Chatg2050** [vercel/community#3795] [Account Issue] [answered]
### Question

When I use github binding to log in to vercel, it prompts, "This user account is blocked.". I am very puzzled. 
I hope it can be resolved as soon as possible, thank you.

**Comment by amyegan** [comment, marked as answer]
Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. For more info, please check this guide: [Why has my account or deployment been paused?](https://vercel.com/guides/why-is-my-account-deployment-blocked)
